lib/manageiq/release/internationalization.rb

```diff
@@ -0,0 +1,104 @@
+module ManageIQ
+  module Release
+    class Internationalization
+
+      def self.instance(options = {})
+        new(options[:branch], :dry_run => options[:dry_run])
```

Comment:
if you change the interface to use keyword args, then you don't need these class methods to manipulate the hashes.  This is how we do it in other classes.

update_i18n.rb:

```ruby
ManageIQ::Release::Internationalization.new(opts.slice(:branch, :dry_run)).update_pot
```

Note that we use `slice` here just to ensure that no "bonus" parameters get sent in.

```
def initialize(branch:, dry_run: true)
```